sql/moz-fx-data-shared-prod/subscription_platform/nonprod_stripe_subscriptions/view.sql

```diff
@@ -77,10 +77,16 @@ invoices_provider_country AS (
   WHERE
     invoices.status = "paid"
 ),
-subscriptions_promotion_codes AS (
+subscriptions_promotions AS (
   SELECT
     invoices.subscription_id,
     ARRAY_AGG(DISTINCT promotion_codes.code IGNORE NULLS) AS promotion_codes,
+    SUM(
+      COALESCE(coupons.amount_off, 0) + COALESCE(
+        CAST((invoices.subtotal * coupons.percent_off / 100) AS INT64),
```

Comment:
Not sure if this should be a `SAFE_CAST` in case any of these values are null?

Reply:
Nulls are fine with normal `CAST()`.  What would be an issue is if there were any incompatible values like non-numeric strings, but here all the values are already numbers and this is just to avoid the calculation resulting in floats.